### ISSUE TITLE
AF2 Using MMseqs ColabFold 1.5.4 crash with ValueError: unknown model_type af2_multimer_v3

### ISSUE THREAD

**Issue by jfbazan** [sokrypton/ColabFold#546]
When I ran an open ColabFold1.5.3 on my browser this morning for a multimer_v3 job, it crashed at the "Run Prediction" stage right after collecting the MSAs and drawing the sequence coverage plot, with an error message signaling ValueError: Unknown model_type alphafold2_multimer_v3. This error recurred when choosing 'multimer_v3' or 'auto' settings. When I restarted a clean AF2, it upgraded to ColabFold1.5.4––a happy surprise!––& I guessed that that ValueError occurred because I was trying to run an outdated ColabFold. So I tried again, first sampling the new "DeepFold v1" choice (how does this compare with Multimer_v3?), and this ran well....but when I tried to purposefully run model_type = auto (or multimer_v3), the program crashed with that same ValueError message, see below. Thanks very much in advance for tracking down the bug, & looking forward to using the new ColabFold version!

2023-12-28 17:52:48,081 Setting max_seq=508, max_extra_seq=2048
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
[<ipython-input-30-06098a2c69e9>](https://localhost:8080/#) in <cell line: 65>()
     63 
     64 download_alphafold_params(model_type, Path("."))
---> 65 results = run(
     66     queries=queries,
     67     result_dir=result_dir,

2 frames
[/content/colabfold/alphafold/models.py](https://localhost:8080/#) in get_model_haiku_params(data_dir, model_type, model_number, use_fuse, to_jnp)
     35         is_deepfold = True
     36     else:
---> 37         raise ValueError(f"Unknown model_type {model_type}")
     38 
     39     path = os.path.join(data_dir, "params", file)

**ValueError: Unknown model_type alphafold2_multimer_v3**

**Comment by milot-mirdita** [comment]
Could you please reload the notebook and try again? The issue should be gone now.

**Comment by jfbazan** [comment]
The patch seems to work––multimer_v3 option runs just fine! However, one problem remains: the final **PAE plots** for the top 5 models do not show, and one only gets the sequence coverage & lDDT plots. I noticed that the (initially working) DeepFold_v1 models also did not show their PAE plots, but wasn't sure they'd be implemented yet.

**Comment by sokrypton** [comment]
Unfortunately, DeepFold_v1 was not trained to return pAE.

**Comment by milot-mirdita** [comment]
@jfbazan the PAE plots for multimer are gone too?

**Comment by jfbazan** [comment]
Indeed, DeepFold_v1 doesn't divulge PAE (yet;), while AF2's multimer_3 still does...but reveals them only in a .png file that's part of the zipped download. In short, it fails to show PAE plots as part of the 3 types of viewable plots (PAE, seq coverage, lDDT) for the models at the end of the prediction run.

**Comment by milot-mirdita** [comment]
I am confused, I do see the PAE plots in colab. Which notebook does not show the PAE plots for multimer?

**Comment by milot-mirdita** [comment]
Can you upload a screenshot please?

**Comment by jfbazan** [comment]
After reloading the new ColabFold, have not experienced the missing PAE plot on my end, chalk it up to weird browser behavior. Thx for your patience, again...& HNY!
